operator/docs/lokistack/sop.md

```diff
@@ -136,3 +136,161 @@ A service(s) has crashed.
 
 - Check the logs of the service that is panicking
 - Examine metrics for signs of failure
+
+## Loki Request Latency
+
+### Impact
+
+A service(s) is affected by slow request responses.
+
+### Summary
+
+A service(s) is slower than expected at processing data.
+
+### Severity
+
+`Critical`
+
+### Access Required
+
+- Console access to the cluster
+- Edit access to the deployed operator and Loki namespace:
+  - OpenShift
+    - `openshift-logging` (LokiStack)
+    - `openshift-operators-redhat` (Loki Operator)
+
+### Steps
+
+- Check the logs of all the services
+- Check to ensure that the Loki components can reach the storage
+
+## Loki Tenant Rate Limit
+
+### Impact
+
+A tenant is being rate limited, resulting in potential loss of data.
+
+### Summary
+
+A service(s) is rate limiting at least 10% of all incoming requests.
+
+### Severity
+
+`Warning`
+
+### Access Required
+
+- Console access to the cluster
+- Edit access to the deployed operator and Loki namespace:
+  - OpenShift
+    - `openshift-logging` (LokiStack)
+    - `openshift-operators-redhat` (Loki Operator)
+
+### Steps
+
+- Examine the metrics for the reason and tenant that is being limited: `loki_discarded_samples_total{namespace="<namespace>"}`
+- Change the ingestion limits for the affected tenant or decrease the rate of logs entering the system
```

Comment:
Links to the CRD limits spec would be useful and maybe a small table between the reasons and the limits config knobs. For example:
- For reason `rate_limited` you can play with `IngestionRate` and/or `IngestionBurstSize`
- For reason `stream_limit` you can play with `MaxGlobalStreamsPerTenant`
- For query limits you can play with `MaxEntriesLimitPerQuery` or `MaxChunksPerQuery` or `MaxQuerySeries`

Reply:
I've added a table for the `ingester`. However, for the `querier`, there doesn't seem to be any `reason` for those in the metrics. I'd imagine this is probably because it's meant to be expressed via the response maybe?